addons/website/views/snippets.xml

```diff
@@ -1107,8 +1107,7 @@
 
     <!-- Carousel | .s_carousel | .s_quotes_carousel -->
     <div data-js="Carousel"
-        data-selector="section"
-        data-target="> .carousel">
```

Comment:
What is this fixing ?

Reply:
converts .carousel selectors to .s_carousel to prevent these options from showing up on gallery.

Reply:
Except that gallery has no `section > .carousel` but a `section > .container > .carousel` so I don't think it is fixing anything?

Reply:
You are right about that but still the selector is I think better with `.s_carousel` than with `section > .carousel` We want it to target the snippet carousel and not all carousel that are child of a section. Leading to potential issues

Reply:
What kind of issue?

Reply:
If another "new snippet's carousel" fits the selector and has it's own options it will have both the s_carousel's options and it's options. s_carousel's options might not be working properly on the "new snippet's carousel" like it is the case for `.carousel-item` at the moment. In the image gallery `.carousel-item` have both image gallery's  and s_carousel's options.

Reply:
The carousel item one is a fix. This one is not, it is an option to toggle a standard bootstrap class on the carousel element, so it should work on any carousel, including the gallery image one. If the gallery image one does not work with it, it would be a bug of the gallery snippet. In any case, no useless change in non-master versions, so this has to be removed.